en/docs/develop/restapis/tenant-management.yaml

```diff
@@ -451,6 +456,10 @@ components:
           type: string
           example: kim
           description: Last name of the owner.
+        provisioningMethod:
+          type: string
+          example: inline-password
+          description: If the provisioning method is inline-password then a valid password should be sent in the request body, if the provisioning method is invite-via-email then password doesn't need to be send through request body, instead an emai link will be sent to the given email address to set the password.
```

Comment:
Would be better to format the description to more clearly state the applicable values.